package.json

```diff
@@ -31,64 +31,68 @@
   },
   "dependencies": {
     "@expo-google-fonts/source-sans-pro": "0.2.2",
-    "@expo/cli": "^0.4.11",
-    "@react-native-async-storage/async-storage": "1.17.7",
+    "@expo/cli": "0.4.11",
+    "@react-native-async-storage/async-storage": "1.17.11",
     "@react-native-community/masked-view": "0.1.11",
     "@react-navigation/native": "6.0.11",
     "@react-navigation/stack": "6.2.2",
     "@rneui/base": "4.0.0-rc.5",
     "@rneui/themed": "4.0.0-rc.5",
-    "expo-application": "4.1.0",
-    "expo-constants": "13.1.1",
-    "expo-device": "4.2.0",
-    "expo-font": "10.1.0",
-    "expo-image-picker": "13.1.1",
-    "expo-linear-gradient": "11.3.0",
-    "expo-linking": "^3.1.0",
-    "expo-permissions": "13.2.0",
-    "expo-splash-screen": "0.15.1",
-    "expo-status-bar": "1.3.0",
-    "expo-updates": "0.13.3",
+    "expo": "47.0.13",
+    "expo-application": "5.0.1",
+    "expo-constants": "14.0.2",
+    "expo-device": "5.0.0",
+    "expo-font": "11.0.1",
+    "expo-image-picker": "14.0.2",
+    "expo-linear-gradient": "12.0.1",
+    "expo-linking": "3.1.0",
```

Comment:
after installing I get the following warning. Does it work with that version?

`expo-linking@3.1.0 - expected version: ~3.3.0`